lib/logic/common/plan_instance_cubit.dart

```diff
@@ -86,6 +85,7 @@ class PlanInstanceCubit extends ReloadableCubit {
 		List<UITaskInstance> uiInstances = await _taskInstancesService.mapToUIModels(updatedTaskInstances);
 
 		Future.wait(updates);
+		uiPlanInstance = await _dataAggregator.loadPlanInstance(planInstance: planInstance);
```

Comment:
It's not going to fetch the fields again unless you only pass it the `planInstanceId` param - so right now the duration should be wrong